python/cudf_polars/cudf_polars/experimental/benchmarks/pdsh.py

```diff
@@ -1097,6 +1103,12 @@ def _query_type(query: int | str) -> list[int]:
     default=False,
     help="Use RMM async memory resource.",
 )
+parser.add_argument(
+    "--rapidsmpf-oom-protection",
+    action=argparse.BooleanOptionalAction,
+    default=False,
+    help="Use rapidsmpf CUDA managed memory-based OOM protection.",
```

Comment:
nitpick
```suggestion
    help="Use Rapidsmpf CUDA managed memory-based OOM protection.",
```

Reply:
I decided to leave it like that because we have other entries like `help="Use rapidsmpf for general spilling.",`. Technically the right name would be `RapidsMPF` (project) or `rapidsmpf` (module name), I'd argue `Rapidsmpf` is the least appropriate, but the other lines starts with the word `Rapidsmpf` and that's likely why it was capitalized. We should probably review naming in this file later.